uninstall.cmake

```
@@ -19,6 +19,23 @@
 # Add custom rule to uninstall the package.
 #
 macro(_SETUP_PROJECT_UNINSTALL)
+  # Detect if the .catkin was created previously
+  if(NOT DEFINED PACKAGE_CREATES_DOT_CATKIN
+     OR NOT "${PACKAGE_PREVIOUS_INSTALL_PREFIX}" EQUAL
```

```suggestion
     OR NOT "${PACKAGE_PREVIOUS_INSTALL_PREFIX}" STREQUAL
```
(`EQUAL` is only used to compare numbers)

---

done!